Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -3896,6 +3898,14 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
 #endif
 }
 
+void UnifiedPDFPlugin::resetActiveAnnotationIfNeeded()
+{
+#if PLATFORM(MAC)
+    if (m_activeAnnotation)
+        std::exchange(m_activeAnnotation, nullptr)->commit();
```

Comment:
Not really a fan of the clever exchange/call a function on the same line. Also, I'd prefer if this share code with `setActiveAnnotation()`.

Reply:
Sure, I can address both of these things!

Reply:
My idea was to not muddy the logic in `setActiveAnnotation()` with a cleanup path that ignores `isFullFramePlugin()` for the teardown case.

Reply:
Sharing code with `setActiveAnnotation()` addresses both comments. https://github.com/WebKit/WebKit/commit/5de44a38d40f53c25418ec5eac373de9098aa6f9